src/eopf_geozarr/data_api/__init__.py

```diff
@@ -43,6 +43,6 @@
     "Sentinel2QualityGroup",
     "Sentinel2ReflectanceGroup",
     "Sentinel2ResolutionDataset",
-    "Sentinel2Root",
+    "Sentinel2ARoot",
```

Comment:
We should not make any distinction in the Sentinel A, B or C platform. There is no difference, they are just twins.